### PR TITLE
init_project

### PR DESCRIPTION
**Pull request by trungduc97**
Ready

### CODE REVIEW

**Comment by trungduc97** [comment]
Ready

**Comment by trungduc97** [comment]
Ready

**Comment by trungduc97** [comment]
Ready

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/node_modules
+/yarn-error.log
+
+/public/assets
+.byebug_history
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+/database.yml
```
Comment (thanhhungchu95) [inline .gitignore]: `/config/database.yml` instead
and run command: `git rm -r --cached config/database.yml`

**Comment by trungduc97** [comment]
Ready

**Pull request merged**